packages/driver/src/util/commandAUTCommunication.ts

```diff
@@ -0,0 +1,11 @@
+export const isRunnerAbleToCommunicateWithAut = (autWindow?: Window): boolean => {
+  const win = autWindow || Cypress.state('window')
+
+  try {
+    win.location.href
```

Comment:
if the AUTwindow is undefined this throws, but also if win.location is cross-origin, except when chrome web security is off? I swear you have this documented some place in the PR and I wonder if it is good to add to the function here as documentation.